.github/workflows/android-ci.yml

```diff
@@ -6,18 +6,35 @@ on:
     tags-ignore:
       - '**'
     paths-ignore:
+      - ".gitignore"
+      - "README.md"
+      # ignore other platforms
+      - "platform/darwin/**"
+      - "platform/glfw/**"
       - "platform/ios/**"
+      - "platform/macos/**"
+      - "platform/node/**"
       - "platform/qt/**"
-      - ".github/**"
-      - "README.md"
-      - ".gitignore"
+      # ignore CI for other platforms
+      - ".github/FUNDING.yml"
+      - ".github/workflows/android-ci-pull.yml"
+      - ".github/workflows/android-docker*.yml"
+      - ".github/workflows/android-release.yml"
+      - ".github/workflows/gh-pages*.yml"
+      - ".github/workflows/ios*.yml"
+      - ".github/workflows/macos*.yml"
+      - ".github/workflows/node*.yml"
+      - ".github/workflows/qt*.yml"
+      - ".github/workflows/update-gl-js.yml"
+
+concurrency:
+  # cancel jobs on PRs only
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
```

Comment:
Since we have android-ci-pull.yml and android-ci.yaml, and since we want to only cancel workflow runs on pull requests, I wonder if this file should not have concurrency settings at all?